libs/features/webapp-prompt/tooltip.css

```diff
@@ -0,0 +1,50 @@
+[data-pep-dismissal-tooltip]::after {
+  content: attr(data-pep-dismissal-tooltip);
+  position: absolute;
+  display: inline-flex;
+  z-index: 201;
+  background-color: #3B63FB;
+  height: fit-content;
+  width: 8.875rem;
+  top: 120%; 
+  right: -75%;
+  word-break: break-word;
+  border-radius: 0.4375rem;
+
+  padding-inline-start: 0.5625rem;
+  padding-inline-end: 0.5625rem;
+  padding-block-start: 0.25rem;
+  padding-block-end: 0.3125rem;
+  
+  transition: opacity 0.5s;
+
+  font-family: 'Adobe Clean';
+  font-size: 0.75rem;
+  line-height: 0.9375rem;
+  color: white;
+  pointer-events: none;
+}
+
+[data-pep-dismissal-tooltip]::before { 
+  content: '';
+  position: absolute;
+  z-index: 200;
+  background-color: #3B63FB;
+  width: 0.44rem;
+  height: 0.44rem;
+  border-radius: 0.05rem;
+  left: calc(50% - 0.22rem);
+  top: 110%;
+  pointer-events: none;
+  
+  transform: rotate(45deg);
+
+  transition: opacity 0.5s;
+}
+
+@media (min-width: 1520px) {
```

Comment:
This is a non-standard media query value, not found anywhere else throughout the project. Is it really necessary? Could the `left` value not be defined in some other way?

Reply:
Excepting cases where there's an icon to the right of the Unav (which, afaict, is exceedingly rare), we don't have enough space for the tooltip if it's centered unless the viewport is 1520px or greater. So the tooltip is off center when there's not enough space. See [here](https://www.figma.com/proto/xbe8zXEg1LWmTl0PCzhck4/PEP?page-id=0%3A1&node-id=1324-4136&t=R6HV7GiYLkDIwlvh-0&scaling=scale-down-width&starting-point-node-id=2%3A848)
The tooltip is 160px (10rem, including the padding -- from the designs), and we begin to get more space to the right at 1440px. So including half the waffle icon and the profile icon (16+32) and the gap between icons (4px/0.25rem) and the 8px feds gutter, we have 60px; 1440+60. The extra 20 px is for a little extra space to the right.
The reason we need to do this is because the tooltip has `absolute` positioning to position it relative to the waffle icon. So it can't know when it might be cut off by the viewport. So we need this non standard query to know when it's okay to center the tooltip.